staging/src/k8s.io/apiserver/pkg/server/healthz/healthz.go

```diff
@@ -107,6 +115,13 @@ func InstallPathHandler(mux mux, path string, checks ...HealthzChecker) {
 	klog.V(5).Info("Installing healthz checkers:", formatQuoted(checkerNames(checks...)...))
 
 	mux.Handle(path, handleRootHealthz(checks...))
+	// TODO: Consider removing this in a future version. This basically adds a bunch
```

Comment:
I'm ok with hiding them.  I'm not ok with removing the ability to get detailed information about each health check.

Reply:
As discussed offline, I'll remove the TODO as it is an orthogonal change.